changelogs/unreleased/gh-9505-relay-stack-use-after-return.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/config
```

Comment:
Why the `/config` subsystem?

It should be `bugfix/replication` or probably `bugfix/core`